lcx/package.json

```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "scripts": {
     "prepublishOnly": "yarn build && yarn test:unit",
-    "build": "tsc",
+    "build": "yarn && tsc",
```

Comment:
Please take a look at [Clean adapter build process #135](https://github.com/smartcontractkit/external-adapters-js/pull/135).

Reply:
Very nice, reverted this change.